src/external/timeAgo.js

```diff
@@ -1,39 +1,17 @@
-var timeparts = [
-    { name: 'millenni', div: 31556736000, p: 'a', s: 'um' },
-    { name: 'centur', div: 3155673600, p: 'ies', s: 'y' },
-    { name: 'dec', div: 315567360 },
-    { name: 'yr', div: 31556736 },
-    { name: 'month', div: 2629728 },
-    { name: 'day', div: 86400 },
-    { name: 'hr', div: 3600 },
-    { name: 'min', div: 60 },
-    { name: 'sec', div: 1 },
-]
+import dayjs from 'dayjs';
 
-function timeAgo(comparisonDate1, comparisonDate2) {
-    var i = 0,
-        parts = [],
-        interval = Math.floor(
-            (comparisonDate2.getTime() - comparisonDate1.getTime()) / 1000
-        )
-    for (; interval > 0; i += 1) {
-        let value = Math.floor(interval / timeparts[i].div)
-        interval = interval - value * timeparts[i].div
-        if (value) {
-            parts.push(
-                value +
-                    ' ' +
-                    timeparts[i].name +
-                    (value !== 1 ? timeparts[i].p || 's' : timeparts[i].s || '')
-            )
-        }
-    }
-    if (parts.length === 0) {
-        return '00:00:00'
-    }
-    return parts.join(', ')
-}
+export default function timeAgo (date1, date2) {
+    let from = dayjs(date1), to = dayjs(date2);
+    let parsed = {};
+    for (let field of ['year', 'month', 'day', 'hour', 'minute', 'second']) 
+        parsed[field] = (to.diff(from, field))
 
-export default timeAgo
+    let out = '';
+    if (parsed.day % 30) out += `${parsed.day % 30}:`
```

Comment:
Fix the `% 30` please, day is not necessary to be cut

Reply:
Resolved.